src/Mutator/IgnoreMutator.php

```diff
@@ -82,10 +82,10 @@ public function canMutate(Node $node): bool
             return false;
         }
 
-        /** @var ReflectionClass|false $reflectionClass */
-        $reflectionClass = $node->getAttribute(ReflectionVisitor::REFLECTION_CLASS_KEY, false);
+        /** @var ClassReflection|null $reflectionClass */
+        $reflectionClass = $node->getAttribute(ReflectionVisitor::REFLECTION_CLASS_KEY);
 
-        if ($reflectionClass === false) {
+        if (!$reflectionClass instanceof ClassReflection) {
```

Comment:
`!($reflectionClass instanceof ClassReflection)`? I always find the `!` horribly confusing in this scenario